### DISCUSSION TITLE
SSL options needed in localhost environment?

### DISCUSSION THREAD

**Discussion by muhaimincs** [planetscale/discussion#87] [FAQs] [answered]
I have to include this in connectionOption:

```
ssl  : {
      ca : process.env.PLANETSCALE_SSL_CERT_PATH
    },
```

**Comment by jgreet** [comment, marked as answer]
Yes, depending on the language or framework you're using, you may need to set a certificate path for any environment that you want to connect to PlanetScale from. This includes both local dev environments and production environments and the path can be different in each.

For example, in the development environment for an app on my Mac the correct path is `/etc/ssl/cert.pem` but if I deploy that same app to Vercel I'll need to make sure it's set to `/etc/pki/tls/certs/ca-bundle.crt` there.

More on secure database connections can be found in our documentation:
https://docs.planetscale.com/reference/secure-connections
